integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -80,10 +80,28 @@ def single_learner_assessment_grade_transmit(self, exporter, **kwargs):
                 enterprise_customer_uuid,
                 lms_user_id,
                 learner_data.course_id,
-                f'create_assessment_reporting started for enrollment {learner_data.enterprise_course_enrollment_id}'
+                'create_assessment_reporting started for '
+                f'integrated_channel_enterprise_enrollment_id={learner_data.enterprise_course_enrollment_id}'
             ))
 
             serialized_payload = learner_data.serialize(enterprise_configuration=self.enterprise_configuration)
+
+            if self.enterprise_configuration.dry_run_mode_enabled:
+                remote_id = getattr(learner_data, kwargs.get('remote_user_id'))
+                encoded_serialized_payload = encode_data_for_logging(serialized_payload)
+                LOGGER.info(generate_formatted_log(
+                    self.enterprise_configuration.channel_code(),
+                    enterprise_customer_uuid,
+                    lms_user_id,
+                    learner_data.course_id,
+                    'dry-run mode '
+                    'skipping create_assessment_reporting for enrollment '
+                    f'integrated_channel_enterprise_enrollment_id={learner_data.enterprise_course_enrollment_id}, '
+                    f'integrated_channel_remote_user_id={remote_id}, '
+                    f'integrated_channel_serialized_payload_base64={encoded_serialized_payload}'
```

Comment:
log the payload json as a base64 encoded blob we can pull out when parsing dry run logs